int64 opt1
int64 opt2
int64 opt3
float64[6] joints
geometry_msgs/Pose[] poses
